src/components/ProductList/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ItemList from 'components/ItemList';
+
+function ProductList({ values }) {
```

Comment:
I left a comment about the nesting for the other pair here: https://github.com/the-collab-lab/tcl-18-smart-shopping-list/pull/20/files#r567335341

Curious to know why you two also chose this strategy :)?

Reply:
aaaah good call, it's cause the other team already fixed it. It was about the nested divs: https://github.com/the-collab-lab/tcl-18-smart-shopping-list/pull/20/files/310b41bfa50e7f6b4498ae8c1ad4cbb5da515094#r567335341

I think this one will work! else, here's a handy screenshot: 
![image](https://user-images.githubusercontent.com/54291087/106387350-64fe2e80-63a7-11eb-95bf-dc3e322a27ad.png)